SLHCUpgradeSimulations/Configuration/python/postLS1Customs.py

```diff
@@ -100,9 +102,14 @@ def customise_DQM(process):
     # Turn off flag of gangedME11a
     process.l1tCsctf.gangedME11a = cms.untracked.bool(False)
     # Turn off "low bias voltage" region in HCAL noise filters
-    # and switch the default decision
     if hasattr(process,'HBHENoiseFilterResultProducer'):
         process.HBHENoiseFilterResultProducer.IgnoreTS4TS5ifJetInLowBVRegion = cms.bool(False)
+    return process
+
+
+def customise_DQM_25ns(process):
+    # Switch the default decision of the HCAL noise filter
+    if hasattr(process,'HBHENoiseFilterResultProducer'):
         process.HBHENoiseFilterResultProducer.defaultDecision = cms.string("HBHENoiseFilterResultRun2Loose")
     return process
```

Comment:
This gives me an error, presumably because the function customise_DQM_25ns is
used/called before it is defined.

```
Creating RECO+VALIDATION+DQM GRun_MC
RAW2DIGI,L1Reco,RECO,VALIDATION,DQM,ENDJOB
entry file:RelVal_DigiL1RawHLT_GRun_MC.root
Step: RAW2DIGI Spec: 
Step: L1Reco Spec: 
Step: RECO Spec: 
Step: VALIDATION Spec: 
 in preparing validation
Step: DQM Spec: 
Step: ENDJOB Spec: 
# Conditions read from  CMS_CONDITIONS  via FrontierProd 
customising the process with Base from HLTrigger/Configuration/CustomConfigs
customising the process with customisePostLS1 from SLHCUpgradeSimulations/Configuration/postLS1Customs
Traceback (most recent call last):
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw/CMSSW_7_5_X_2015-04-23-2300/bin/slc6_amd64_gcc491/cmsDriver.py", line 43, in <module>
    run()
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw/CMSSW_7_5_X_2015-04-23-2300/bin/slc6_amd64_gcc491/cmsDriver.py", line 15, in run
    configBuilder.prepare()
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw/CMSSW_7_5_X_2015-04-23-2300/python/Configuration/Applications/ConfigBuilder.py", line 2188, in prepare
    self.pythonCfgCode += self.addCustomise()
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw/CMSSW_7_5_X_2015-04-23-2300/python/Configuration/Applications/ConfigBuilder.py", line 869, in addCustomise
    self.process=getattr(package,fcn)(self.process)
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw/CMSSW_7_5_X_2015-04-23-2300/python/SLHCUpgradeSimulations/Configuration/postLS1Customs.py", line 53, in customisePostLS1
    process = customise_DQM_25ns(process)
NameError: global name 'customise_DQM_25ns' is not defined
```